src/engraving/playback/playbackeventsrenderer.cpp

```diff
@@ -64,18 +64,24 @@ void PlaybackEventsRenderer::renderNoteEvents(const Ms::Chord* chord, const mpe:
         return;
     }
 
-    PlaybackContext ctx;
-    ctx.nominalPositionTick = chord->tick().ticks();
-    ctx.nominalDurationTicks = chord->durationTypeTicks().ticks();
-    ctx.beatsPerSecond = chord->score()->tempomap()->tempo(ctx.nominalPositionTick).val;
-    ctx.nominalTimestamp = timestampFromTicks(chord->score(), ctx.nominalPositionTick);
-    ctx.nominalDuration = durationFromTicks(ctx.beatsPerSecond, ctx.nominalDurationTicks);
-    ctx.nominalDynamicLevel = nominalDynamicLevel;
-    ctx.profile = profile;
+    int chordPosTick = chord->tick().ticks();
+    int chordDurationTicks = chord->durationTypeTicks().ticks();
+    BeatsPerSecond bps = chord->score()->tempomap()->tempo(chordPosTick).val;
```

Comment:
There is no need to get a val

Reply:
thanks, fixed.

Forgot to remove it when changed type